document/core/binary/conventions.rst

```diff
@@ -20,7 +20,8 @@ Except for a few exceptions, the binary grammar closely mirrors the grammar of t
    Implementations of decoders must support all possible alternatives;
    implementations of encoders can pick any allowed encoding.
 
-The recommended extension for files containing WebAssembly modules in binary format is ":math:`\T{.wasm}`".
+The recommended extension for files containing WebAssembly modules in binary format is ":math:`\T{.wasm}`"
+and the recommended Media Type is ":math:`\T{application/wasm}`".
```

Comment:
Nit: "media type" lower case?

Reply:
https://www.iana.org/assignments/media-types/media-types.xhtml seems to capitalize it everywhere as a proper noun.  I was actually thinking of linking Media Type to https://www.iana.org/assignments/media-types/media-types.xhtml, if you could tell me the right incantation...

Reply:
Interesting, Wikipedia doesn't do that. But that's okay. For linking you can follow the examples in util/macros.def, and then just write `|MimeType|`.